From: David Jarvie <djarvie@kde.org>
Date: Wed, 07 Mar 2012 22:08:50 +0000
Subject: Fix error saving changed alarms when closing Edit Alarm dialogue
X-Git-Url: http://quickgit.kde.org/?p=kdepim.git&amp;a=commitdiff&amp;h=ae66e32fb194e01e06521c78e9c72d772d291261
---
Fix error saving changed alarms when closing Edit Alarm dialogue
---


--- a/kalarm/Changelog
+++ b/kalarm/Changelog
@@ -1,6 +1,10 @@
 KAlarm Change Log
 
-=== Version 2.8.1 --- 19 February 2012 ===
+=== Version 2.8.2 --- 7 March 2012 ===
+- [Akonadi] Fix error saving changed alarms when closing Edit Alarm dialogue.
+- [Akonadi] Show old-format calendars in read-only colour in calendar list.
+
+=== Version 2.8.1 (KDE 4.8.1) --- 19 February 2012 ===
 - [Akonadi] Don't give option to save new alarms in old format calendars.
 - [Akonadi] Prevent duplicate prompts to update format of new calendar resource.
 - [Akonadi] Automatically disable duplicated calendar resources.

--- a/kalarm/akonadimodel.cpp
+++ b/kalarm/akonadimodel.cpp
@@ -1284,7 +1284,7 @@ KAEvent AkonadiModel::event(const Item&
     if (e.isValid())
     {
 
-        Collection c = data(index, ParentCollectionRole).value<Collection>();
+        Collection c = data(ix, ParentCollectionRole).value<Collection>();
         // Set collection ID using a const method, to avoid unnecessary copying of KAEvent
         e.setCollectionId_const(c.id());
         if (collection)

--- a/kalarm/kalarm.h
+++ b/kalarm/kalarm.h
@@ -28,7 +28,7 @@
 #else
 #define VERSION_SUFFIX "-res"
 #endif
-#define KALARM_VERSION "2.8.1" VERSION_SUFFIX
+#define KALARM_VERSION "2.8.2" VERSION_SUFFIX
 
 #define KALARM_NAME "KAlarm"
 #define KALARM_DBUS_SERVICE  "org.kde.kalarm"  // D-Bus service name of KAlarm application

